client/src/Movies/MovieList.js

```diff
@@ -48,7 +49,7 @@ function MovieDetails({ movie }) {
         <div key={star} className="movie-star">
           {star}
         </div>
-      ))}
+      ))}</Link>
```

Comment:
and here, give your closing tag some room to breathe for the sake of readability:
```javascript
  ))}
</Link>
```